ycmd/tests/subcommands_test.py

```diff
@@ -24,17 +24,20 @@
                           PathToTestFile,
```

Comment:
I don't have a preference to how we organize them as long as it results in files that _don't_ have 1k+ lines. :)

Your suggestion sounds reasonable.